fr/core-libraries/caching.rst

```diff
@@ -38,10 +38,10 @@ votre propre système de mise en cache. Les moteurs de cache intégrés sont:
   est une extension PHP qui fournit des fonctionnalités similaires à APC.
 * ``MemcachedEngine`` Utilise l'extension
   `Memcached <http://php.net/memcached>`_. Il est aussi une interface avec
-  memcache mais il fournit une meilleur performance.
+  Memcached mais il fournit une meilleur performance.
```

Comment:
This sentence doesn't make sense anymore and should be removed I think. It's coming from 2.x doc where MemacheEngine was an option.
http://book.cakephp.org/2.0/en/core-libraries/caching.html